.github/workflows/publish.yml

```
@@ -3,21 +3,27 @@ name: PyPI Publish
 on:
   release:
     types: [created]
+  workflow_dispatch:
+    inputs:
+      test_release:
+        description: If true, publish to test.pypi.org
+        required: true
+        default: true
+        type: boolean
 
 jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python
-        uses: actions/setup-python@v5
+      - name: Install uv & Python
+        uses: astral-sh/setup-uv@v6
         with:
-          python-version: '3.x'
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install poetry
-          poetry install
-      - run: poetry publish --build
+          enable-cache: true
+          cache-dependency-glob: "uv.lock"
+          python-version-file: "pyproject.toml"
+      - run: uv build
+      - run: uv publish
         env:
-          POETRY_PYPI_TOKEN_PYPI: ${{secrets.POETRY_PYPI_TOKEN_PYPI}}
+          UV_PUBLISH_TOKEN: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.test_release && secrets.TEST_PYPI_API_TOKEN || secrets.POETRY_PYPI_TOKEN_PYPI }}
+          UV_PUBLISH_URL: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.test_release && https://test.pypi.org/legacy/ || https://upload.pypi.org/legacy/ }}
```

Shouldn't this be quoted ?


```suggestion
          UV_PUBLISH_URL: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.test_release && 'https://test.pypi.org/legacy/' || 'https://upload.pypi.org/legacy/' }}
```